pyproject.toml

```diff
@@ -68,8 +71,8 @@ markers = "slow: Mark the test as slow. Can be skipped with --skip_slow"
 
 [tool.poetry.plugins]
 [tool.poetry.plugins."console_scripts"]
-"manim" = "manim.__main__:main"
-"manimce" = "manim.__main__:main"
+"manim" = "manim.__main__:manim"
+"manimce" = "manim.__main__:manim"
```

Comment:
for me `main` seems to look better

Reply:
This would change the tests in `test_commands.py` see response https://github.com/ManimCommunity/manim/pull/1013#discussion_r598964444